src/lib/components/Card/Card.stories.tsx

```diff
@@ -45,3 +46,17 @@ WithDecorativeImage.storyName = 'With decorative image';
 WithDecorativeImage.args = {
   imgSrc: 'https://flowbite.com/docs/images/blog/image-1.jpg',
 };
+
+export const WithNextImage = Template.bind({});
+WithNextImage.storyName = 'With Next.js Image component';
+WithNextImage.args = {
+  renderImage: () => (
+    <Image
+      alt="Meaningful alt text for an image that is not purely decorative"
+      loader={({ src }) => src}
```

Comment:
The loader prop here is important. If we do not provide this, `Image` returns some weird "ExoticComponent" and the Storybook crashes.

Reply:
This is only to satisfy the storybook, right?

Reply:
No. This is a hack to make `Image` from `next` actually return a `ReactNode`. If you do not provide this, `Image` will return some funky object that makes the code crash. I guess it has something to do with an additional build step with `nextjs` where `nextjs` picks up these objects and finalizes them by injecting the proper loader. If we provide it here, everything works and the image gets rendered (without any optimizations of course). I will add a comment to explain this funkyness.

Reply:
```suggestion
      {/* Necessary to make `Image` render a React Node */}
      loader={({ src }) => src}
```